openslides/users/static/templates/user/user-form.html

```diff
@@ -1,8 +1,80 @@
-<h1 ng-if="user.id">{{ user.get_short_name() }}</h1>
-<h1 ng-if="!user.id">{{ gettext("New User") }}</h1>
+<h1 ng-if="user.id">{{ gettext("Edit participant") }}</h1>
+<h1 ng-if="!user.id">{{ gettext("New participant") }}</h1>
 
-<form>
-    firstname: <input type="text" ng-model="user.first_name"><br>
-    lastname: <input type="text" ng-model="user.last_name"><br>
-    <input type="submit" ng-click="save(user)" value="Save" />
+<div id="submenu">
+    <a ng-href="user" class="btn btn-sm btn-default">
+        <i class="fa fa-angle-double-left fa-lg"></i>
+        {{ gettext('Back to overview') }}
+    </a>
+</div>
+
+<form name="userForm">
+    <div class="form-group" ng-class="{ 'has-error has-feedback' : userForm.inputUsername.$error.required }">
+        <label for="inputUsername">{{ gettext("Username") }}</label>
+        <input type="text" ng-model="user.username" ng-required="true" class="form-control" name="inputUsername">
+        <span ng-if="userForm.inputUsername.$error.required" class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+        <p ng-show="userForm.inputUsername.$error.required" class="error-message help-block">
+            {{ gettext("Username is required.") }}
+        </p>
+    </div>
+    <div class="form-group row">
+        <div class="col-xs-2">
+            <label for="inputTitle">{{ gettext("Title") }}</label>
+            <input type="text" ng-model="user.title" class="form-control" name="inputTitle">
+        </div>
+        <div class="col-xs-5">
+            <label for="inputFirstName">{{ gettext("First name") }}</label>
+            <input type="text" ng-model="user.first_name" class="form-control" name="inputFirstName">
+        </div>
+        <div class="col-xs-5">
+            <label for="inputLastName">{{ gettext("Last name") }}</label>
+            <input type="text" ng-model="user.last_name" class="form-control" name="inputLastName">
+        </div>
+    </div>
+    <div class="form-group">
+        <label for="inputStructureLevel">{{ gettext("Structure level") }}</label>
+        <input type="text" ng-model="user.structure_level" class="form-control" name="inputStructureLevel">
+    </div>
+    <div class="form-group" ng-class="{ 'has-error has-feedback' : userForm.selectGroups.$error.required }">
```

Comment:
Proposal: remove the whitespaces after '{' and ':' and before '}'. This is a normal dict.